content/docs/accessibility.md

```diff
@@ -35,20 +35,21 @@ Note that all `aria-*` HTML attributes are fully supported in JSX. Whereas most
   aria-required="true"
   onChange={onchangeHandler}
   value={inputValue}
-  name="name"
+  name="անուն"
 />
 ```
 
-## Semantic HTML {#semantic-html}
-Semantic HTML is the foundation of accessibility in a web application. Using the various HTML elements to reinforce the meaning of information
-in our websites will often give us accessibility for free.
+## Իմաստավոր HTML {#semantic-html}
 
```

Comment:
էս ավել տող ա

Reply:
45?

Reply:
42| ## Semantic HTML {#semantic-html}
43| Semantic HTML is the foundation of...

այ էս դառել ա սենց

42| ## Իմաստավոր HTML {#semantic-html}
43|